app/__init__.py

```diff
@@ -4,8 +4,10 @@
 from sqlalchemy import create_engine, Column, Integer, String, func
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
+from flask_babel import Babel
 
 app = Flask(__name__)

```

Comment:
@olegpidsadnyi  I need **app** and **babel** variables in module context. Because later they will be imported in **views** module.